src/server/apollo.ts

```diff
@@ -0,0 +1,93 @@
+import * as Sentry from '@sentry/node';
+import express from 'express';
+import http from 'http';
+import { expressMiddleware } from '@apollo/server/express4';
+import { ApolloServer } from '@apollo/server';
+import { ApolloServerPluginDrainHttpServer } from '@apollo/server/plugin/drainHttpServer';
+import { buildSubgraphSchema } from '@apollo/subgraph';
+import { ApolloServerPluginLandingPageGraphQLPlayground } from '@apollo/server-plugin-landing-page-graphql-playground';
+import { ApolloServerPluginLandingPageDisabled } from '@apollo/server/plugin/disabled';
+import { sentryPlugin, errorHandler } from '@pocket-tools/apollo-utils';
+import config from '../config';
+import { ContextManager } from './context';
+import { readClient } from '../database/client';
+import {
+  initItemEventHandlers,
+  itemsEventEmitter,
+  snowplowEventHandler,
+  sqsEventHandler,
+  unifiedEventHandler,
+} from '../businessEvents';
+import queueDeleteRouter from './routes/queueDelete';
+import { BatchDeleteHandler } from '../aws/batchDeleteHandler';
+import { EventEmitter } from 'events';
+import { initAccountDeletionCompleteEvents } from '../aws/eventTypes';
+import { typeDefs } from './typeDefs';
+import { resolvers } from '../resolvers';
+
+export async function startServer(port: number) {
```

Comment:
Nice, I was thinking almost the exact same thing here. Also TIL port 0 dynamically allocates a port.

Do we want to make this accept an entire config object as a parameter and pass it through in main? We'd be just a few steps away from being able to run tests in parallel.

Reply:
Sure yeah it's a possibility, but it would only work for any tests that didn't use the shared db backend (so query tests only). I don't really have a strong opinion.

Reply:
It's definitely out of scope for this PR, but the SQL database names and dynamodb table names are in the config file. If we randomly generated those, migrated, seeded, and then cleaned up as a part of the test lifecycle it would be possible here as well.  Then you could probably run the integration tests in watch mode as well while developing.

Reply:
Yeah it's a cool idea and I think worthwhile to consider for future repos, particularly ones that actually are sagas and have control of the db. Just thinking of cost-benefit for this particular repo